src/openrct2/interface/Colour.h

```diff
@@ -213,8 +213,8 @@ constexpr uint8_t kColourNumNormal = 54;
 
 static constexpr uint8_t kLegacyColourFlagTranslucent = (1 << 7);
 
-#define TEXT_COLOUR_254 (254)
-#define TEXT_COLOUR_255 (255)
+constexpr int TEXT_COLOUR_254 = 254;
+constexpr int TEXT_COLOUR_255 = 255;
```

Comment:
Please apply these two changes as well:
- Change type from `int` to `colour_t`
- Rename to `kTextColour254` and `kTextColour255`

(Of course, make sure it compiles locally before pushing, this requires changes in a few files.)